spreadsheet/formula/expression.go

```diff
@@ -10,4 +10,6 @@ package formula
 type Expression interface {
 	Eval(ctx Context, ev Evaluator) Result
 	Reference(ctx Context, ev Evaluator) Reference
+	String() string
```

Comment:
What kind of string is this?  Is this similar to the "formula string" that one sees in excel in the formula bar when a cell is selected?  Needs to be documented better or add examples.  Need to know if this is just for "pretty printing" when debugging or if this string actually has a specific format.